cmd/kubepug.go

```diff
@@ -144,12 +186,17 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&format, "format", "stdout", "Format in which the list will be displayed [stdout, plain, json, yaml]")
 	rootCmd.PersistentFlags().StringVar(&filename, "filename", "", "Name of the file the results will be saved to, if empty it will display to stdout")
 	rootCmd.PersistentFlags().StringVar(&inputFile, "input-file", "", "Location of a file or directory containing k8s manifests to be analysed")
+	rootCmd.PersistentFlags().BoolVar(&monitor, "monitor", true, "run kubepug as a persistant prometheus server to monitor deprocations")
```

Comment:
s/deprocations/deprecations

Also, can we make this defaulting to false to keep the original behavior of the program, and allow existing users to opt-in to the feature?